docusaurus.config.js

```diff
@@ -208,6 +208,10 @@ const redirectsList = [
     from: "/docs/HyperSync/hyperrpc-url-endpoints",
     to: "/docs/HyperSync/hyperrpc-supported-networks",
   },
+  {
+    from: '/',
+    to: '/docs/HyperIndex/overview',
```

Comment:
Maybe it should redirect to https://envio.dev ? I personally expected the behavior every time I clicked on the logo 😅

Reply:
Yes, I agree with this :+1:

Reply:
Thanks! Agreed. 

So we are saying docs.envio.dev "/" redirects to envio.dev?

Reply:
Agree the logo should take us back to the site. 

But was thinking docs.envio.dev should just redirect to the hyperindex overview page?

Reply:
Ah, we are actually talking about different things. I didn't look closely.

I think this redirect makes sense. But when clicking the 'envio' logo in the top left, it should go to the landing site.

Reply:
This code isn't working though :grimacing: https://envio-docs-git-change-landing-docs-page-envio-9ec2d675.vercel.app/

No redirects happening.

Reply:
Yes I'm not sure why I haven't managed to get it working :( Fixing again now